xun/functions/function.py

```diff
@@ -58,11 +59,44 @@ def __init__(self, desc, dependencies, max_parallel):
         self.max_parallel = max_parallel
         self.hash = Function.sha256(desc, dependencies)
         self._graph_builder = None
+        self._code = None
 
     @property
     def name(self):
         return self.desc.name
 
+    @property
+    def code(self):
+        class FunctionCode:
```

Comment:
Make this class a nested class inside function

Reply:
Done